cmake/IgnManualSearch.cmake

```diff
@@ -42,17 +42,32 @@
 # PATH_SUFFIXES: Optional. Parameter forwarded to the find_path and find_library calls.
 #
 macro(ign_manual_search package)
+  # TODO(chapulina) Enable warnings after all libraries have migrated.
+  # message(WARNING "ign_manual_search is deprecated, use gz_manual_search instead.")
 
-  #------------------------------------
-  # Define the expected arguments
   set(options INTERFACE)
   set(oneValueArgs "TARGET_NAME")
   set(multiValueArgs "HEADER_NAMES" "LIBRARY_NAMES" "PATH_SUFFIXES")
-
-  #------------------------------------
-  # Parse the arguments
   _gz_cmake_parse_arguments(gz_manual_search "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
+  set(gz_manual_search_skip_parsing true)
+  gz_manual_search(${PACKAGE_NAME})
```

Comment:
hm... I think this should be `${package}` instead of `${PACKAGE_NAME}` (see https://github.com/gazebosim/gz-common/issues/386)